iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/scalable/GeneratorScalable.java

```diff
@@ -24,6 +24,8 @@ class GeneratorScalable extends AbstractInjectionScalable {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(GeneratorScalable.class);
 
+    private boolean allowOutOfBoundsTargetP = false;
```

Comment:
Can you use the `ScalingParameters` class to define the parameter?

Reply:
Yes. I have change this